package.json

```diff
@@ -21,7 +21,9 @@
     "eslint-plugin-import": "2.7.0",
     "eslint-plugin-jsx-a11y": "5.1.1",
     "eslint-plugin-react": "7.1.0",
+    "jest": "20.0.4",
```

Comment:
you shouldn't need jest. Should already be a part of create-react-app

Reply:
Uninstalled Jest